src/ios/CameraSessionManager.m

```diff
@@ -49,7 +49,7 @@ - (void) setupSession:(NSString *)defaultCamera completion:(void(^)(BOOL started
                 BOOL success = TRUE;
                 
                 NSLog(@"defaultCamera: %@", defaultCamera);
-                if ([defaultCamera isEqual: @"front"]) {
+                if ([defaultCamera isEqual: @"front"] || [options[@"direction"] isEqual: @"front"]) {
```

Comment:
same as for the android part, we can refactor this method
setupsession takes defaultCamera, maybe we can remove it here and include it in options directly, and in the method we only deal with options, and we put options as the first parameter of the setupsession, @HashirRajah what do you think ?